lecture09.tex

```diff
@@ -36,18 +36,21 @@ \section{Lecture 9}
 We \emph{define}
 \[ \Norm_{L|K}(\Pfrak) := \pfrak^{f(\Pfrak|\pfrak)}, \]
 and extend it to a homomorphism of abelian groups on $J_{L}(B)$ in the unique way.
-This norm map is clearly multiplicative, and \todo{transitive in towers.}{Add a precise assertion.}
+This norm map is clearly multiplicative, and transitive in towers.
+Precisely, suppose $K \subset E \subset L$ is a tower of finite separable field extensions and $A \subset B \subset B$ the integral closure of $A$ in $E$ and $L$, respectively.
+Let $\mathfrak p$ be a prime of $A$, $\mathfrak q$ a prime of $B$ lying above $\mathfrak p$ and $\mathfrak P$ a prime of $C$ lying above $\mathfrak q$.
+Then $f(\mathfrak{P} / \mathfrak{p})=f(\mathfrak{P} / \mathfrak{q}) f(\mathfrak{q} / \mathfrak{p})$, which implies that $\Norm_{L|K}(\mathfrak P) = \Norm_{E|K}(\Norm_{L|E}(\mathfrak P))$.
```

Comment:
The transitivity of inertia degrees is not really related to transitivity of the norm maps.